unison-src/transcripts/fix1578.md

```diff
@@ -9,7 +9,7 @@ This transcript shows how suffix-based name resolution works when definitions in
 As setup, we'll add a data type `Day` and a definition `foo.bar : Nat`.
 
 ```unison:hide
-unique type Day = Sun | Mon | Tue | Wed | Thu | Fri | Sat | Sun
+unique type Day = Sun | Mon | Tue | Wed | Thu | Fri | Sat
```

Comment:
Looks like this error snuck past the old logic; it's caught at parsing time now 👍🏼